### ISSUE TITLE
Set 'info' row option

### ISSUE THREAD

**Issue by vredesbyyrd** [OmarCastro/rofi-blocks#20]
Hi, thanks for sharing this great tool. 

When using rofi's [script modi](https://github.com/davatorium/rofi/blob/next/doc/rofi-script.5.markdown), since version 1.6 you can pass some additional 'row options'. One being `info`. 

- info: Info that, on selection, gets placed in the ROFI_INFO environment variable. This entry does not get searched.

Its passed the same way icons are. 

Having this functionality (or something similar that is not visible in the list but can be parsed by the user after selection) in rofi would be another way to separate data from presentation, as requested in #14. 

**A use case:** 

I am making a music browser, and it would be nice to do something like this: 

```
{
  "name": "select entry",
  "value": "The label of the entry",
  "info": 144
}
```
 Upon selection, only output the info key, which in the case would be the `artist_id` or `album_id`.

**Comment by OmarCastro** [comment]
Hello, this hint gave me a good idea how to solve #14 ( thought about it, but then forgot due to life responsibilities ) 

I plan to use "data" property to define the row metadata, used as additional data to communicate with the external application, solving the data/presentation division. This property will only allow strings, if not defined, it will be an empty string, since any info can be converted to a string

For input info I will add the following properties {{data}} and {{data_escaped}}, which contains the data defined on the row

in short, an entry will be possible to define like this:
```js
{
  "name": "select entry",
  "value": "The label of the entry",
  "data": "artist_id: 1\nalbum_id: 1" //<-- yaml data, but you can use whatever format you wish
}
```

and the default input format will be

```js
{"name":"{{name_escaped}}", "value":"{{value_escaped}}", "data": "{{data_escaped}}"}
```
meaning you will receive the same info as input in this case

**Comment by vredesbyyrd** [comment]
Thanks for the response, and right on, that sounds like a perfect approach imho.

**Comment by vredesbyyrd** [comment]
Just gave 21eb4059ff049d0788c41fdf5485478c4d9aa5e0 a test drive and the data property works as advertised, perfect! 

Much appreciate you implementing this, and thanks again for the very useful rofi plugin.
